ipld/read.go

```diff
@@ -2,15 +2,29 @@ package ipld
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/ipfs/go-cid"
 	ipld "github.com/ipfs/go-ipld-format"
 
 	"github.com/celestiaorg/celestia-node/ipld/plugin"
 	"github.com/celestiaorg/nmt"
+
 	"github.com/celestiaorg/nmt/namespace"
 )
 
+// ErrByzantine is an error converted from rsmt2d.ByzantineRow/Col +
+// Merkle Proof of each share.
+type ErrByzantine struct {
+	Index  uint8
+	Shares []*NamespacedShareWithProof
+	IsRow  bool
```

Comment:
Would be nice to rely on the enum defined in the new version of rsmt2d. The one is called `Axis`. IsRow feels kinda ugly, but the new version of rsmt2d is breaking the existing code.

Reply:
Will add a todo and create an issue because in our version rsmt2d `row` and `col` are lowercase.

Reply:
Yes pls